plugins/notifications-backend/README.md

```diff
@@ -59,7 +59,7 @@ User parameter is mandatory because it is needed for message status and filterin
 Request:
 
 ```bash
-curl 'http://localhost:7007/api/notifications/notifications?user=loggedinuser&pageNumber=0&pageSize=0'
+curl 'http://localhost:7007/api/notifications/notifications?user=loggedinuser&read=0&pageNumber=0&pageSize=0'
```

Comment:
Is there a reason to use numbers instead of boolean values (or their strings `"true", "false"` for the `read`? The `readByUser` or the DB is boolean-like.

Reply:
yes. i could not find a javascript or typescript function that takes a string representation of boolean and returns the boolean value. if you have one then please share. if we can not find one then I think 0/1 is the easiest.

Reply:
I would not drive API definition by that.

There is none such converter natively. I believe there will be a few implementations in npm libraries.
Anyway, we can go with something like:
```
    const toBoolean = (str?: string) => {
      if (!str) {
        return undefined;
      }
      return str === 'true' ? true : false;
    }
```

Reply:
done